### PR TITLE
[SWAGGER] Document Users endpoints

### PR DESCRIPTION
**Pull request by ledelaespriella**
## Jira Ticket

- [WA-40](https://n2-alkemy-sku-31-10.atlassian.net/jira/software/projects/WA/boards/1?assignee=6183439e26a54000711fbace&selectedIssue=WA-40)

## Description
AS: member of project
I Want: see how to use endpoints of users
To: correctly apply each of the endpoints

## Evidence:

Captura de Pantalla 2022-11-11 a la(s) 9 50 29 a m

## GET

Captura de Pantalla 2022-11-11 a la(s) 9 55 53 a m

## POST

Captura de Pantalla 2022-11-11 a la(s) 9 59 51 a m

Captura de Pantalla 2022-11-11 a la(s) 10 00 13 a m

### Whit bad request:

Captura de Pantalla 2022-11-11 a la(s) 10 04 43 a m

## GET BY ID

Captura de Pantalla 2022-11-11 a la(s) 10 07 53 a m

### Whit bad request:

Captura de Pantalla 2022-11-11 a la(s) 10 08 08 a m

## PUT

Captura de Pantalla 2022-11-11 a la(s) 10 10 15 a m

Captura de Pantalla 2022-11-11 a la(s) 10 10 50 a m

### Whit bad request:

Captura de Pantalla 2022-11-11 a la(s) 10 11 18 a m

## DELETE

Captura de Pantalla 2022-11-11 a la(s) 10 12 07 a m

### Whit bad request:

Captura de Pantalla 2022-11-11 a la(s) 10 42 18 a m

### CODE REVIEW

File: routes/users.router.js [hunk 1]
```diff
@@ -40,13 +40,221 @@ const router = express.Router();
...
+ *                     message: Users obtained successfully
+ *                     body:
+ *                        firstName: Mandy
+ *                        lastName: Pyford
+ *                        email: mpyford0@xinhuanet.com
+ *                        createdAt: 2022-11-10T21:45:49.000Z       
+ *   post:
+ *     summary: Create user
+ *     tags: [User]
+ *     description: This can only be done by the logged in user.
```
Comment (germanWalton) [inline routes/users.router.js]: Correct this please. This endpoint is for register an user, you don't have to be logged to do this.
You can put  Register an user

File: routes/users.router.js [hunk 2]
```diff
@@ -40,13 +40,221 @@ const router = express.Router();
...
+ *       '500':
+ *         description: Internal Server Error
+ *   put:
+ *     summary: Edit user by Id
+ *     tags: [User]
+ *     parameters:
+ *        - name: id
+ *          in: path
+ *          required: true
+ *          description: ID of user to return.
```
Comment (germanWalton) [inline routes/users.router.js]: It must be ID of the user to edit

File: routes/users.router.js [hunk 3]
```diff
@@ -40,13 +40,221 @@ const router = express.Router();
...
+ *       '500':
+ *         description: Internal Server Error
+ *   delete:
+ *     summary: Delete user by Id
+ *     tags: [User]
+ *     parameters:
+ *        - name: id
+ *          in: path
+ *          required: true
+ *          description: ID of user to return.
```
Comment (germanWalton) [inline routes/users.router.js]: It must be ID of the user to delete

**Review by germanWalton — CHANGES_REQUESTED** [review]

**Comment by germanWalton** [comment]
Missing to make the login endpoint that belongs to users. Please do it.

**Review by germanWalton — APPROVED** [review]

**Review by AgustinCanteros — APPROVED** [review]

**Pull request merged**